README.rst

```diff
@@ -15,9 +15,72 @@ Package skeleton for a PureSkill.gg PySkill.
 Description
 -----------
 
-Bootstrap a new Python_ PureSkill.gg PySkill in less than a minute.
+Bootstrap a new Python PureSkill.gg PySkill in less than a minute.
 
-.. _Python: https://www.python.org/
+🎓 Start with the Tutorial!
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This repository is both a template for creating new a PySkill,
+and fully functioning tutorial project.
+
+⚡️💡⚡️ **If you are just getting started with PureSkill.gg data science, start here!**.
+
+Follow these steps to start the tutorial.
+On completing the tutorial, you will understand how to do data science
+with PureSkill.gg data and be setup to download data from the public data set.
+Once you complete the tutorial, you may use this repository to bootstrap you own project.
+
+Requirements
+^^^^^^^^^^^^
+
+- Linux, macOS, or Windows.
```

Comment:
```suggestion

```
They better not try this on a TI-84 calculator. "This was tested on windows, mac, and linux." is probably sufficient.